### PR TITLE
Add build scripts for packaging tendrl-monitoring-integration

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
Signed-off-by: Timothy Asir J <tjeyasin@redhat.com>

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
@r0h4n , @nthomas-redhat , @mbukatov , @dahorak,  @rishubhjain , Please review

**Comment by TimothyAsirJeyasing** [comment]
https://github.com/Tendrl/specifications/issues/178

File: tendrl-monitoring-integration.spec [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+Version: 1.0.0
+Release: 1%{?dist}
+BuildArch: noarch
+Summary: Module for Tendrl Monitoring Integration
+Source0: %{name}-%{version}.tar.gz
+License: LGPLv2+
+URL: https://github.com/Tendrl/monitoring-integration
+
+#Requires: tendrl-commons
+Requires: grafana
```
Comment (cybernth) [inline tendrl-monitoring-integration.spec:11]: Requires:
graphite-web
python-carbon 
python-whisper

File: tendrl-monitoring-integration.spec [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+# remove the sphinx-build leftovers
+rm -rf html/.{doctrees,buildinfo}
+
+%install
+%{__python} setup.py install --single-version-externally-managed -O1 --root=$RPM_BUILD_ROOT --record=INSTALLED_FILES
+install -m  0755  --directory $RPM_BUILD_ROOT%{_var}/log/tendrl/monitoring_integration
+install -m  0755  --directory $RPM_BUILD_ROOT%{_sysconfdir}/tendrl/monitoring_integration
+#install -Dm 0644 monitoring-integration.service $RPM_BUILD_ROOT%{_unitdir}/monitoring-integration.service
+install -Dm 0644 tendrl/monitoring_integration/grafana/grafana.ini $RPM_BUILD_ROOT%{_sysconfdir}/grafana/grafana.ini
+install -Dm 0644 graphite.conf.sample.yml $RPM_BUILD_ROOT%{_sysconfdir}/tendrl/monitoring_integration/graphite.conf.yml
```
Comment (cybernth) [inline tendrl-monitoring-integration.spec:34]: Note that these config files might be required here:

https://github.com/Tendrl/performance-monitoring/blob/master/etc/tendrl/performance-monitoring/graphite-web.conf.sample

https://github.com/Tendrl/performance-monitoring/blob/master/etc/tendrl/performance-monitoring/carbon.conf.sample

stay tuned to https://github.com/Tendrl/monitoring-integration/pull/1

File: tendrl-monitoring-integration.spec [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+%post
+%preun
```
Comment (cybernth) [inline tendrl-monitoring-integration.spec:37]: Same as above. refer https://github.com/Tendrl/performance-monitoring/blob/master/tendrl-performance-monitoring.spec

File: tendrl-monitoring-integration.spec [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+%postun
```
Comment (cybernth) [inline tendrl-monitoring-integration.spec:38]: same as above

**Review by cybernth — CHANGES_REQUESTED** [review]

**Comment by mbukatov** [comment]
I guess that this could be closed, as I see the rpm spec file included in master branch of Tendrl/monitoring-integration, right?

**Pull request closed without merging**